test/functional/tests/data_integrity/test_data_integrity_5d.py

```diff
@@ -0,0 +1,194 @@
+            fs_utils.create_directory(mount_point)
+        TestRun.LOGGER.info(f"Create filesystem {filesystem} on {core.system_path}")
+        core.create_filesystem(filesystem)
+        TestRun.LOGGER.info(f"Mount filesystem {filesystem} on {core.system_path} to "
+                            f"{mount_point}")
+        core.mount(mount_point)
+        sync()
+
+    return core
+
+
+def prepare_devices():
+    cache_device = TestRun.disks['cache']
+    core_device_1 = TestRun.disks['core1']
+    core_device_2 = TestRun.disks['core2']
+    core_device_3 = TestRun.disks['core3']
+    core_device_4 = TestRun.disks['core4']
+
+    cache_device.create_partitions([Size(50, Unit.GibiByte), Size(50, Unit.GibiByte),
+                                    Size(50, Unit.GibiByte), Size(50, Unit.GibiByte)])
+    core_device_1.create_partitions([Size(150, Unit.GibiByte)])
+    core_device_2.create_partitions([Size(150, Unit.GibiByte)])
+    core_device_3.create_partitions([Size(150, Unit.GibiByte)])
+    core_device_4.create_partitions([Size(150, Unit.GibiByte)])
+    sync()
+
+    return cache_device, core_device_1, core_device_2, core_device_3, core_device_4
+
+
+def prepare_instances_5d():
+    cache_device, core_device_1, core_device_2, core_device_3, core_device_4 = prepare_devices()
+
+    instances = [start_instance(cache_device.partitions[0], core_device_1.partitions[0],
+                                CacheMode.WT),
+                 start_instance(cache_device.partitions[1], core_device_2.partitions[0],
+                                CacheMode.WB),
+                 start_instance(cache_device.partitions[2], core_device_3.partitions[0],
+                                CacheMode.WA),
+                 start_instance(cache_device.partitions[3], core_device_4.partitions[0],
+                                CacheMode.WO)]
+
+    return instances
+
+
+def prepare_instances_5d_dss():
+    cache_device, core_device_1, core_device_2, core_device_3, core_device_4 = prepare_devices()
+
+    fs_list = [Filesystem.ext3, Filesystem.ext4, Filesystem.xfs]
+    random.shuffle(fs_list)
+    instances = [start_instance(cache_device.partitions[0], core_device_1.partitions[0],
+                                CacheMode.WT, fs_list[0]),
+                 start_instance(cache_device.partitions[1], core_device_2.partitions[0],
+                                CacheMode.WB, fs_list[1]),
+                 start_instance(cache_device.partitions[2], core_device_3.partitions[0],
+                                CacheMode.WA, fs_list[2]),
+                 start_instance(cache_device.partitions[3], core_device_4.partitions[0],
+                                CacheMode.WO, fs_list[0])]
+
+    return instances
+
+
+def run_workload(instances):
+    fio_run = Fio().create_command()
+    fio_run.io_engine(IoEngine.libaio)
+    fio_run.direct()
+    fio_run.time_based()
+    fio_run.do_verify()
+    fio_run.verify(VerifyMethod.meta)
```

Comment:
According to the documentation, this method is deprecated:
https://fio.readthedocs.io/en/latest/fio_doc.html

Consider using another verification method here and in run_workload_dss()

Reply:
Done.